oap-server/server-receiver-plugin/pom.xml

```diff
@@ -45,6 +45,7 @@
         <module>configuration-discovery-receiver-plugin</module>
         <module>skywalking-event-receiver-plugin</module>
         <module>skywalking-zabbix-receiver-plugin</module>
+        <module>skywalking-ebpf-receiver-plugin</module>
```

Comment:
The module name looks too general, what about renaming to `skywalking-rover-receiver-plugin`?

Reply:
It is following protocol naming rule, https://github.com/apache/skywalking-data-collect-protocol/tree/master/ebpf/profiling